gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -14,7 +14,10 @@ var addHiddenGbItemsCallback = function (hiddenItems) {
 
     hiddenItems.forEach(i => {
 
-      $(".gb-filter :input:checked[value='" + i + "']")
+      $(".gb-filter :input:checked")
+        .filter(function() {
+          return $(this).attr("value") === i;
+        })
```

Comment:
i think pref here would be just get rid of jQuery entirely while you're at it. absolutely no need for it anymore

Reply:
You're welcome to revise this. This is the fix we're going with locally next week before I leave for about a month, and I've other priorities to care for before then. Otherwise, I can resume taking a look at this when I'm back in April.